helmfile/values/networkpolicy-common.yaml.gotmpl

```diff
@@ -32,3 +34,14 @@ kured:
 
 velero:
   enabled: {{ and .Values.velero.enabled .Values.networkPolicies.velero.enabled }}
+
+certManager:
+  enabled: {{ .Values.networkPolicies.certManager.enabled }}
+  namespaces:
+    {{- if eq .Environment.Name "workload_cluster" }}
+    {{- concat .Values.user.namespaces .Values.networkPolicies.certManager.namespaces | toYaml | nindent 4 }}
```

Comment:
Will `.Values.user.namespaces` work for HNC @viktor-f?

Reply:
Good point, no they will not. Unless we start to propagate network policies to child namespaces.
Right now I don't think there is a good way to solve that except for propagating it with hnc or telling our customers that they need to create it themselves if the want ingresses with certs.